pkg/cmd/root.go

```diff
@@ -65,7 +66,9 @@ func init() {
 	rootCmd.PersistentFlags().StringVarP(&nodeLabels,
 		"node-labels", "", "", "labels to filter nodes with")
 	rootCmd.PersistentFlags().StringVarP(&namespaceLabels,
-		"namespace-labels", "n", "", "labels to filter namespaces with")
+		"namespace-labels", "", "", "labels to filter namespaces with")
+	rootCmd.PersistentFlags().StringVarP(&namespace,
+		"namespace", "n", "", "only this namespace (default: all)")
```

Comment:
Tiny nit, the default felt slightly out of place with the other defaults that were automatically generated for help output here. Maybe something like this would be slightly clearer.
```suggestion
		"namespace", "n", "", "only include pods from this namespace")
```